scripts/build/build.js

```diff
@@ -49,16 +45,14 @@ function getBabelConfig(bundle) {
       require.resolve("./babel-plugins/transform-custom-require")
     );
   }
-  if (bundle.transpile) {
-    const targets = { node: 4 };
-    if (bundle.target === "universal") {
-      // From https://jamie.build/last-2-versions
-      targets.browsers = [">0.25%", "not ie 11", "not op_mini all"];
-    }
-    config.presets = [
-      [require.resolve("@babel/preset-env"), { targets, modules: false }]
-    ];
+  const targets = { node: 4 };
+  if (bundle.target === "universal") {
+    // From https://jamie.build/last-2-versions
+    targets.browsers = [">0.25%", "not ie 11", "not op_mini all"];
```

Comment:
Can we keep the IE11 support? People still use it.

Reply:
Has it ever worked properly in IE11? Not sure anyone's tested that. Which usually means no.